deployment/Dockerfile.fedora.amd64

```diff
@@ -1,4 +1,4 @@
-FROM fedora:33
+FROM fedora:latest
```

Comment:
I may not be able to comment on the rest of the changes, as distro packaging is not my strong suite nor focus area, however PLEASE NEVER USE `latest` IN CI!

```suggestion
FROM fedora:36
```

Reply:
I second this, chasing down bugs cause the build scripts silently changed a version without anyone knowing won't be fun, it's better to have an explicit pr for the versions, so that someone could have a chance of knowing what's up.

Reply:
Alright fair enough, I can edit that commit during the rebase, not a big deal.
Ideally the packages would be built in `mock` inside the docker container (as discussed above) and then the base image doesn't matter all that much anyway.

Reply:
Understood, yet still, having versions of dependencies change unexpectedly in any CI environment is just asking for trouble, even if the RPM build itself may be encapsulated in `mock`.

Regardless, could you please rebase the PR, so that is ready for a final review?
We should get it into the next 10.8.x then :smiley:

Reply:
Rebased and pushed, waiting for the CI to complain :)